library-best-practices/JLBP-9.md

```diff
@@ -0,0 +1,14 @@
+[JLBP-9] Ensure consumers always use a higher or equal minimum Java version
```

Comment:
I suggest rewording as, "Support your consumers' minimum Java version."

I still don't like the term consumer here. Maybe "clients' minimum Java version" if you don't like dependents? To me the word "consumer" implies that the library is being used up somehow, and that's not the case.